f8a_worker/workers/repo_dependency_finder.py

```diff
@@ -0,0 +1,175 @@
+"""Output: List of direct and indirect dependencies."""
+
+import requests
+import traceback
+
+from f8a_worker.base import BaseTask
+from f8a_worker.errors import TaskError
+from f8a_worker.graphutils import GREMLIN_SERVER_URL_REST
+from f8a_worker.workers.mercator import MercatorTask
+from f8a_worker.workers.dependency_parser import GithubDependencyTreeTask
+
+
+class RepoDependencyFinderTask(BaseTask):
+    """Finds out direct and indirect dependencies from a given github repository."""
+
+    _mercator = MercatorTask.create_test_instance(task_name='RepoDependencyFinderTask')
+
+    def execute(self, arguments=None):
+        """Task code.
+
+        :param arguments: dictionary with task arguments
+        :return: {}, results
+        """
+        self.log.debug("Arguments passed from flow: {}".format(arguments))
+        self._strict_assert(arguments.get('service_token'))
+
+        github_repo = arguments.get('github_repo')
```

Comment:
@samuzzal-choudhury - can we strip the arguments here
`github_repo = arguments.get('github_repo').strip()`